app/code/Magento/Directory/Setup/Patch/Data/AddMissingCountries.php

```diff
@@ -0,0 +1,90 @@
+<?php
+/**
```

Comment:
Can you rename the file to be more specific, like the countries are you adding. [Take a look](https://github.com/magento/magento2/tree/2.3-develop/app/code/Magento/Directory/Setup/Patch/Data).

Reply:
Hi @osrecio, done